### PR TITLE
Added a Background Music

### PR DESCRIPTION
**Pull request by naveentherocker**
Added a Background Music along with the Youtube_Audio_Library.md
This Song has no Copyright Issues as this song was given by Youtube for the Normal Use.
Ping @niccokunzmann

### CODE REVIEW

File: audio/music/Youtube_Audio_Library.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You can use the YouTube Audio Library to get free music and sound effects to use in your videos. Learn more about how to move viewers in your videos with sound. You can download and use some of our high-quality 320kbps audio tracks and sound effects royalty-free.
```
Comment (niccokunzmann) [inline audio/music/Youtube_Audio_Library.md]: Please follow the documentation about adding a music and 

- [ ] create a license file, this is not a license file but a vision
- [ ] find out the license(s) of the video and copy the whole text into the file

Comment (naveentherocker) [reply audio/music/Youtube_Audio_Library.md]: @niccokunzmann This is a common license file for all the Audios provided by Youtube Audio Library. This has no Copyright Issues https://www.youtube.com/audiolibrary/music

Comment (abishekvashok) [reply audio/music/Youtube_Audio_Library.md]: @naveentherocker Idk about this:
> YouTube cannot give you information about legal liability or other issues that can occur off the platform. You may wish to consult a qualified attorney if you have questions about your use of music.

https://support.google.com/youtube/answer/3376882?hl=en

But we can't get it's license:
https://productforums.google.com/forum/#!topic/youtube/w6Wm4AqYxCs

**Comment by niccokunzmann** [comment]
To make it clear:

> You can use the YouTube Audio Library to get free music and sound effects to use in your videos.

You can also use it to get non-free music. This is not excluded. So, Getting the music with the library does not mean that it has the license of the library.

> You can download and use some of our high-quality 320kbps audio tracks and sound effects royalty-free.

Yes, but which audio files are these? What does royalty-free mean? In Germany, and other legislations, you can not give no license as you can not drop all your rights. You are always the creator, no matter what happens.
"some files can be downloaded all of which are royalty-free" does not exclude that we can not do otherwise.

**Comment by niccokunzmann** [comment]
Also, reading the youtube page: I only see that the standard-youtube-license provides the rights to youtube, not to us. https://www.youtube.com/static?template=terms

**Comment by niccokunzmann** [comment]
Also, if I read the terms of use, they state that there might be additional terms such as naming like in CC-BY, so I think this is not the whole license text.

**Comment by naveentherocker** [comment]
@niccokunzmann Please check this https://youtube-creators.googleblog.com/2013/09/free-music-for-your-youtube-videos.html . It is mentioned that `150 royalty-free instrumental tracks you can use for free, forever, for any creative purpose (not just YouTube videos).`

**Comment by niccokunzmann** [comment]
Cool, so please provide the license file with the source and retrieval date and all is fine. I see this is a blog post with a vision but not a license.

**Pull request closed without merging**